Bottom: 1c9395c4d15227803cc220b719a4cc052dff2d13
Top:    412a65555052c1d9c94a4573b2f2a044b74ad7ec
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-05-10 18:06:22 +0100

Make --enable-gl default when building the GTK HID


---

diff --git a/configure.ac b/configure.ac
index b451980..08ffa53 100644
--- a/configure.ac
+++ b/configure.ac
@@ -386,8 +386,12 @@ fi
 AC_MSG_CHECKING([for whether to use GL drawing])
 AC_ARG_ENABLE([gl],
 [  --enable-gl           Enable GL drawing (with GTK HID)],
-[],[enable_gl=no])
-
+[],[
+	case " $with_gui " in
+		*\ gtk\ *) enable_gl=yes;;
+		* ) enable_gl=no;;
+	esac
+])
 AC_MSG_RESULT([$enable_gl])
 AM_CONDITIONAL(USE_GL, test x$enable_gl = xyes)
